src/gameobjects/container/ContainerWebGLRenderer.js

```diff
@@ -29,6 +29,8 @@ var ContainerWebGLRenderer = function (renderer, container, camera, parentMatrix
         return;
     }
 
+    camera.addToRenderList(src);
```

Comment:
What is src here?  I don't see where this value is supposed to be defined?

Should this be container instead?

Reply:
`camera.addToRenderList(src)` is a new method introduced by 3.52.0, other renderer of game object like [Image](https://github.com/photonstorm/phaser/blob/master/src/gameobjects/image/ImageWebGLRenderer.js#L23) also has this command inside.

Reply:
Yes, but what is the value of src here.  It appears to be undefined.

If you look at the Image class you linked, the function accepts `src` as a parameter:
```
var ImageWebGLRenderer = function (renderer, src, camera, parentMatrix)
```

ContainerWebGLRenderer has a different parameter name:

```
var ContainerWebGLRenderer = function (renderer, container, camera, parentMatrix)
```

So you should change this to `camera.addToRenderList(src);`

Reply:
Thanks, I change `src` to `container` in my next submit